src/state-tomo.jl

```diff
@@ -137,7 +137,7 @@ function fit(method::LSStateTomo,
     # TODO: use quad_form instead of vecnorm? Have 1/vars are diagonal quadratic form
     problem = Convex.minimize( LinearAlgebra.norm( (means - method.realpred*[vec(ρr); vec(ρi)]) .* ivars, 2), constraints )
 
-    Convex.solve!(problem, solver)
+    Convex.solve!(problem, SCS.Optimizer)
```

Comment:
@dieris I'm confused why the original code doesn't work. SCS.jl still exports SCSSolver(), and the SCS.jl documentation seems to imply it should work. I agree what Convex.jl has switched to calling SCS.Optimizer. I'm not sure there is really a difference...

Reply:
The original code works for me and this code does not...

Reply:
interesting... julia 1.4?

Reply:
ohhh that's probably why

Reply:
None of this code has been tested on 1.4 and I know there were some issues with bumping Convex versions

Reply:
Maybe we need to lock down the version numbers in Project.toml...

Reply:
I'm using 1.3 and Convex 0.12.6.  I can try to get things to work with 1.4 but it might take me some time.  If you're in a hurry I would use 1.3 and Convex 0.12.6.

Reply:
ah I see. I could roll back if necessary. I've been testing this code and Qlab [tomography](https://github.com/BBN-Q/Qlab.jl/blob/8af6172a817d4fe5fcfc7a8faebfe301d8057233/src/tomography.jl)  on julia 1.4. There are some tests to be fixed though

Reply:
I'll read up on this and figure out what needs to happen tonight.  We should also try to start testing on 1.4 if that's where people are moving.

Reply:
I don't think the documentation on Convex or SCS is for 1.4. This might explain my previous confusion.

Reply:
So... @dieris I just tested on Julia 1.4 and Convex 0.12.3 and unittests are passing at least.  Try `(@v1.4) pkg> add Convex@0.12.3`

Reply:
Seems like 0.12.6 will also work

Reply:
To be clear, it is working with the old syntax.

Reply:
@dieris would you be okay with breaking this out into a new PR or changing the focus of this one to support newer versions of Convex.jl/SCS.jl??